utils/bundles.stanza

```diff
@@ -104,13 +104,14 @@ public pcb-bundle uart (pins:Tuple<UARTPins>) :
       UART-CK : make-pin(`ck)
       UART-DE : make-pin(`de)
       UART-CS : make-pin(`cs)
+      UART-RX : make-pin(`rx)
+      UART-TX : make-pin(`tx)
+
 
 public defn uart ():
   uart([])
```

Comment:
The default bundle for `uart` and `usart` should contain the `tx` and `rx` pins. Once that change is made, in other places in ocdb, you can leave statements like this alone:
``` 
port uart : uart()
```
i.e. you wouldn't have to do this if you leave the default as having `tx` and `rx`:
```
  port uart : uart([UART-RX, UART-TX])
```

Reply:
If you leave these pins as a default, then how do you make them optional? If you instantiate, then they will always be there